crates/client-api/src/routes/identity.rs

```diff
@@ -77,7 +77,7 @@ pub async fn get_identity<S: ControlStateDelegate>(
             (!identities.is_empty()).then_some(GetIdentityResponse { identities })
         }
     };
-    let identity_response = lookup.ok_or(StatusCode::NOT_FOUND)?;
+    let identity_response = lookup.unwrap_or_default();
```

Comment:
nit: it seems like we should return 400 / Bad Request if no `email` is provided?